### PR TITLE
Add full node and edge schemas to Python graph [KAT-5513]

### PR DESCRIPTION
**Pull request by arthurp**
Cherry-pick of https://github.com/KatanaGraph/katana-enterprise/pull/3903

### CODE REVIEW

File: python/test/test_property_graph.py [hunk 1]
```diff
@@ -16,6 +16,23 @@ def test_load(graph):
     assert len(graph.loaded_edge_schema()) == 3
 
 
+def test_schemas(graph):
+    assert len(graph.loaded_node_schema()) == 17
```
Comment (gliga) [inline python/test/test_property_graph.py:20]: why do `loaded` start with 17; should it be 0 initially (or maybe something is happening when before the graph comes here as the argument, which is the drawback of reviewing code without context :).

**Review by gliga — APPROVED** [review]
Nice!

File: python/test/test_property_graph.py (hunk 1 above)
Comment (arthurp) [reply python/test/test_property_graph.py:20]: The test graph has all properties loaded by default. I think mostly for historical reasons, but it doesn't seem like a big issue to me.

Comment (gliga) [reply python/test/test_property_graph.py:20]: That is totally fine.  If somebody updates that eventually this test will fail so they will have to fix, so all is good.

**Review by roshandathathri — APPROVED** [review]
LGTM

**Comment by tylershunt** [comment]
Merged in #997

**Pull request closed without merging**